.github/workflows/python-publish.yml

```
@@ -93,3 +92,25 @@ jobs:
         path: dist/
     - name: Publish distribution 📦 to PyPI
       uses: pypa/gh-action-pypi-publish@03f86fee9ac21f854951f5c6e2a02c2a1324aec7 # v1
+
+  release:
+    runs-on: ubuntu-latest
+    if: ${{ inputs.release_id }}
```

actionlint (1.7.9)

property "release_id" is not defined in object type {}

(expression)